cmd/finch/pass_init_native.go

```diff
@@ -0,0 +1,25 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build linux
+
+package main
+
+import (
+	"github.com/gopasspw/gopass/pkg/pwgen"
+	"github.com/runfinch/finch/pkg/command"
+)
+
+func (pia *passInitAction) initGpgKey() command.Command {
+	passphrase := pwgen.GeneratePassword(passphraseLength, true)
+	ecc := command.NewExecCmdCreator()
+	cmd := ecc.Create(
+		"gpg2", "--batch", "--passphrase", passphrase, "--quick-gen-key", "finch")
```

Comment:
Does this key expire after some default period?

Reply:
key doesnt expire, the key creation is owned by the customer so they would need to remove it.